tests/Makefile

```diff
@@ -130,7 +130,7 @@ dependencies.sha256:
 	@shasum -a 256 $(BUILDDIR)/images/* > $@
 
 clean:
```

Comment:
What about the [fix-removal-of-generated-files](https://github.com/cartesi/machine-emulator/pull/249/commits/fb679a6cda31955301f58f988b5f984b97f71eff#diff-6bb23a60a80d75ac81e61f9ae0518579761646eac951cca846625688fbfd0380) you did in the other PR? Can we also have the changes you needed from that PR in this one?

Reply:
Yes, I'll update it.

Reply:
I removed [fix-removal-of-generated-files](https://github.com/cartesi/machine-emulator/pull/249/commits/fb679a6cda31955301f58f988b5f984b97f71eff#diff-6bb23a60a80d75ac81e61f9ae0518579761646eac951cca846625688fbfd0380) since it is not a bug and only makes sense in the debian package context. I'll move it there instead.